pygmt/src/basemap.py

```diff
@@ -33,16 +33,16 @@
 )
 @kwargs_to_strings(R="sequence", c="sequence_comma", p="sequence")
 def basemap(self, **kwargs):
-    """
+    r"""
     Plot base maps and frames for the figure.
 
     Creates a basic or fancy basemap with axes, fill, and titles. Several
     map projections are available, and the user may specify separate
     tick-mark intervals for boundary annotation, ticking, and [optionally]
     gridlines. A simple map scale or directional rose may also be plotted.
 
-    At least one of the options *frame*, *map_scale*, *rose* or *compass*
-    must be specified.
+    At least one of the options **frame**, **map_scale**, **rose** or
+    **compass** must be specified.
```

Comment:
Following https://github.com/GenericMappingTools/pygmt/blob/master/CONTRIBUTING.md#example-code-standards, we should use code blocks for Python arguments.
```suggestion
    At least one of the options ``frame``, ``map_scale``, ``rose`` or
    ``compass`` must be specified.
```